Config.php

```diff
@@ -34,6 +34,7 @@
 $secret = file_exists(__DIR__ . '/env.php');
 
 if (file_exists($path . 'config.yml')) {
+  require_once ABSPATH . '/vendor/mustangostang/spyc/Spyc.php';
```

Comment:
I think I removed this because I made the assumption that Spyc would be loaded by the root Composer autoloader. See requirements https://github.com/CityOfNewYork/nyco-wp-config#requirements